scripts/e2e/bootstrap_job/clusterctl.sh

```diff
@@ -46,7 +46,7 @@ chmod +x /usr/local/bin/kubectl
 
 # run clusterctl
 echo "test ${PROVIDER_COMPONENT_SPEC}"
-./bin/clusterctl create cluster --existing-bootstrap-cluster-kubeconfig ~/.kube/config -c ./spec/cluster.yml \
+/tmp/clusterctl/clusterctl create cluster --existing-bootstrap-cluster-kubeconfig ~/.kube/config -c ./spec/cluster.yml \
```

Comment:
I still don't know what this /tmp/clusterctl/clusterctl is.  If you're building a container for clusterctl, why not run the container here?

Reply:
the bootstrap_job is launched by bootstrap_job.yml (generated based on bootstrap_job.template), the job spec will run a initContainer for this job, clusterctl is coming from initContainer (a shared volume mounted at /tmp/clusterctl).

Reply:
I now understand what you're doing.  You're using a container as a packaging system instead of using curl or wget to pull down a binary.  Instead, you're using the container to copy it's binary out so you can execute it.  It doesn't fully feel right, but there aren't any rules against it so I guess it's ok.